lntest/itest/lnd_hold_invoice_force_test.go

```diff
@@ -0,0 +1,112 @@
+package itest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/lightningnetwork/lnd/lntest/wait"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/stretchr/testify/require"
+)
+
+// testHoldInvoiceForceClose demonstrates that recipients of hold invoices
+// will not release active htlcs for their own invoices when they expire,
```

Comment:
Looks like the behavior "will not release active htlcs for their own invoices when they expire" is not tested?

Reply:
Not releasing the htlc is what causes the force close that we test in this, so the behaviour is "inherently" tested by demonstrating that we get a force close. We also assert that we have a htlc on our force close on 99, which proves that we force closed with a htlc on our commit.